lib/jekyll/commands/new.rb

```diff
@@ -121,23 +121,23 @@ def scaffold_path
         # unless the user opts to generate a blank blog or skip 'bundle install'.
 
         def after_install(path, options = {})
-          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
-          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
-
           unless options["blank"] || options["skip-bundle"]
             bundle_install path
           end
+
+          Jekyll.logger.info "New jekyll site installed in #{path.cyan}."
+          Jekyll.logger.info "Bundle install skipped." if options["skip-bundle"]
         end
 
         def bundle_install(path)
           Jekyll::External.require_with_graceful_fail "bundler"
           Jekyll.logger.info "Running bundle install in #{path.cyan}..."
           Dir.chdir(path) do
-            if ENV["CI"]
-              system("bundle", "install", "--quiet")
-            else
-              system("bundle", "install")
+            process, output = Jekyll::Utils::Exec.run("bundle", "install")
+            output.to_s.each_line do |line|
+              Jekyll.logger.info("Bundler:".green, line.strip) unless line.to_s.empty?
```

Comment:
Should not we colorize the output rather than `Bundler:`?

![jekyll-new-output](https://cloud.githubusercontent.com/assets/103008/21724746/e8ea4f40-d435-11e6-9c83-cb363da20dc3.png)